src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -401,6 +429,10 @@ private function getErrorMessages(Exception $e): array
                     'When redirecting towards a product you must select a target product.',
                     'Admin.Notifications.Error'
                 ),
+                ProductConstraintException::INVALID_ONLINE_DATA => $this->trans(
+                    'A product without name cannot be set online.',
```

Comment:
```suggestion
                    'Please enter a name for this product.',
```

Reply:
I'm not sure the message is specific enough, because you can leave a product without a name as long as it is still online
It's only when you try to "publish" it without its name that this error pops up, so the online part should be mentioned in the message I think or the merchant won't understand why suddenly this error happens